### PR TITLE
Fixed up instructions and layout.

### PR DESCRIPTION
**Pull request by skelterjohn**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,14 +7,10 @@
 # To build this application:
 
 ```
-$ mkdir wgo
-$ cd wgo
-$ export GOPATH=`pwd`
-$ export GOBIN=$GOPATH/bin
-$ export PATH=$GOBIN:$PATH
 $ go get github.com/skelterjohn/wgo
 $ go install github.com/skelterjohn/wgo
```
Comment (gmlewis) [inline README.md]: Why do we need `go get` followed by `go install`?
According to https://golang.org/cmd/go/, a `go get` also performs an install.

Comment (skelterjohn) [inline README.md]: That's right - my PR to this repo simplifies the instructions quite a bit.

Comment (gmlewis) [inline README.md]: Am I reading this wrong?  Can't the new line 11 be deleted?

Comment (jbuberel) [inline README.md]: @skelterjohn Please keep in mind that these repos are meant for experimental use by Glenn and I. That is why the instructions are so verbose. When we figure out where these examples will eventually live, they'll be cleaned up substantially.

The reason for all of the directory creation and environment variable setting was to demonstrate that this was a completely isolated development workspace/environment.

Comment (skelterjohn) [inline README.md]: Oh, I didn't realize this was a comment on the PR. oops, fixed.

Comment (skelterjohn) [inline README.md]: Then I recommend exporting GOPATH='', GOBIN=''

**Comment by gmlewis** [comment]
LGTM

**Comment by gmlewis** [comment]
Sweet!  I patched this in to my local repo, and it was then trivial to deploy this to production!
Jason, can you please merge John's PR and then I will add my own PR for deployment instructions?

**Pull request merged**